src/modules/sidebar/partials/ShareList.vue

```diff
@@ -12,7 +12,7 @@
 				class="row">
 				<div class="fix-col-2">
 					<div style="display:flex; align-items: center;">
-						<NcAvatar :user="getShareAvatar(share)" :is-no-user="share.receiverType !== 'user'" />
+						<NcAvatar :user="share.receiver" :is-no-user="share.receiverType !== 'user'" />
```

Comment:
```vue
<NcAvatar :user="share.receiver" :display-name="share.receiverDisplayName || share.receiver" :is-no-user="share.receiverType !== 'user'" />
```
So that the default avatar uses the color and letters of the displayname